src/components/Icons/ProductIcon.tsx

```diff
@@ -0,0 +1,26 @@
+import { SvgIcon, SvgIconProps } from '@material-ui/core';
+import * as React from 'react';
+import { FC } from 'react';
+
+export const ProductIcon: FC<SvgIconProps> = (props) => (
```

Comment:
Let's call it a `ScriptIcon` instead or a name that matches exactly what the icon is. 

This is also important here because we'll have multiple types of products, and each type will have their own icon I'm guessing.